pyiceberg/cli/output.py

```diff
@@ -157,7 +157,7 @@ def describe_properties(self, properties: Properties) -> None:
         Console().print(output_table)
 
     def text(self, response: str) -> None:
-        Console().print(response)
+        Console(soft_wrap=True).print(response)
```

Comment:
some `test_console.py` outputs are too long and end up with an extra `\n` in the middle of the string, causing tests to fail